src/Service/MoreInfoService/AbstractMoreInfoService.php

```diff
@@ -610,7 +612,9 @@ private function getDefaultIdentifierInformation(string $isType, string $isIdent
 
         if ($this->provideDefaultCover()) {
             $image = new ImageType();
-            $image->_ = $this->transformer->transform(self::FALLBACK_IMAGE_URL);
+            // The image size here should be the size of the default image or at least the default image should be
+            // bigger then the default transformation.
```

Comment:
A bit cryptic comment. Maybe split into more sentences.

Reply:
It was my best take on it.

Reply:
The default image should be larger or equal in size to the default transform.